encoding.go

```diff
@@ -123,6 +127,27 @@ func parseLeafNode(serialized []byte, depth byte) (VerkleNode, error) {
 	return ln, nil
 }
 
+func parseEoAccountNode(serialized []byte, depth byte) (VerkleNode, error) {
+	var values [NodeWidth][]byte
+	offset := leafStemOffset + StemSize + 2*banderwagon.UncompressedSize
+	values[0] = zero32[:]                                   // 0 version
+	values[1] = serialized[offset : offset+leafBalanceSize] // balance
+	var nonce [32]byte
+	offset += leafBalanceSize
+	copy(nonce[:8], serialized[offset:offset+8])
+	values[2] = nonce[:] // nonce
+	values[3] = emptyCodeHash[:]
+	values[4] = zero32[:] // 0 code size
+	ln := NewLeafNodeWithNoComms(serialized[leafStemOffset:leafStemOffset+StemSize], values[:])
+	ln.setDepth(depth)
+	ln.c1 = new(Point)
+	ln.c1.SetBytesUncompressed(serialized[leafStemOffset+StemSize:leafStemOffset+StemSize+banderwagon.UncompressedSize], true)
+	ln.c2 = &banderwagon.Identity
+	ln.commitment = new(Point)
+	ln.commitment.SetBytesUncompressed(serialized[leafStemOffset+StemSize+banderwagon.UncompressedSize:leafStemOffset+StemSize+banderwagon.UncompressedSize*2], true)
```

Comment:
Nit: would be good to define constants for these `leafEoAC1Offset` and `leafEoACommOffset` to be sure same constants are used for serialization?